src/shipit_uplift/tests/test_coverage.py

```diff
@@ -0,0 +1,247 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from __future__ import absolute_import
+
+import urllib.parse
+import json
+import time
+import os
+
+def test_coverage_supported_extensions(client):
+    '''
+    List supported extensions for coverage analysis through api
+    '''
+
+    resp = client.get('/coverage/supported_extensions')
+    assert resp.status_code == 200
+    data = json.loads(resp.data.decode('utf-8'))
+    assert set(data) == set([
+        "c", "h", "cpp", "cc", "cxx", "hh", "hpp",
+        "hxx", "js", "jsm", "xul", "xml", "html", "xhtml",
+    ])
+
+
+
+def retry_request_for_result(client, url, retries=5, timeout=15):
```

Comment:
You could use the `retry` function from cli_common.utils.

Reply:
I only found a `retry` function under `shipit_code_coverage.utils`, and that's subjected to change. Don't see the `cli_common.utils` module in https://github.com/mozilla-releng/services/tree/master/lib/cli_common

Reply:
Right, #849 was not merged yet :smile: